ikarus/utils/dirichletvalues.hh

```diff
@@ -97,12 +101,27 @@ public:
   }
 
   /**
-   * \brief Function to fix (set boolean values to true or false) of degrees of freedom
+   * \brief Fixes (set boolean value to true) a specific degree of freedom
    *
    * \param i An index indicating the DOF number to be fixed
    */
   void fixIthDOF(typename Basis::MultiIndex i) { dirichletFlagsBackend_[i] = true; }
 
+  /**
+   * \brief Unfixes (set boolean value to false) a specific degree of freedom
+   *
+   * \param i An index indicating the DOF number to be fixed
+   */
+  void unfixIthDOF(typename Basis::MultiIndex i) { dirichletFlagsBackend_[i] = false; }
```

Comment:
Maybe we should merge this function to one where the used provides true or false? Maybe `setSingleDof` the `Ith` is also not so nice maybe.

Reply:
Should we also overload this function like `isConstrained` to take a `std::size_t` as well as a `MultiIndex` ?